td-exception/src/interrupt.rs

```diff
@@ -181,7 +181,9 @@ fn generic_interrupt_handler(stack: &mut InterruptStack) {
         return;
     }
 
-    (CALLBACK_TABLE.lock().table[stack.vector].func)(stack);
+    // Copy the pointer out, release the lock and then call the function.
```

Comment:
please add comment:
As such, it can support to handle an exception (#VE) in an interrupt (timer).